packages/actions/src/makePayloadMetaActionCreator.js

```diff
@@ -0,0 +1,22 @@
+import invariant from 'invariant';
+import { nthArg, isNil } from 'ramda';
+import { isPlainObject } from 'ramda-extension';
+
+import configureActionCreator from './configureActionCreator';
+
+/**
+ * Creates a new binary action creator which will use the first argument as the payload and the second argument as the meta.
+ *
+ * @sig String -> (a, {}) -> {type: String, payload: a, meta: {}}
+ */
+const makePayloadMetaActionCreator = type => {
+	const actionCreator = configureActionCreator(type, nthArg(0), nthArg(1));
+
+	return (payload, meta) => {
+		invariant(!isNil(meta) && isPlainObject(meta), 'Meta must be an object.');
```

Comment:
let's split this into two invariants

first message in !isNil(meta)

```
'You did not pass the meta object to an action creator created by makePayloadMetaActionCreator(' + type + ').'
```

for isPlainObject

```
'Action creator created by makePayloadMetaActionCreator(' + type + ') expects the meta argument to be a plain object. Instead, it received ' + meta + '.'
```